OneCommunityGlobal/HighestGoodNetworkApp
# Jingyi unassign team members from tasks

# Description
Gives the user permission to UNASSIGN tasks from only their TEAM members through the Dashboard -> task -> red X. 

## Related PRS (if any):
This frontend PR is related to the #XXX backend PR.
To test this backend PR you need to checkout the #XXX frontend PR.
…

## Main changes explained:
- Create a new permission label 'unassignUserFromTasks"
- use dispatch to check the this permission for conditional rendering in React

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as owner user
5. In the permission management section, grant a volunteer account without permission "unassign team members from tasks".
6. log in to this volunteer account and in the dashboard -> tasks. Check if this volunteer can interact with red X, to unassign user from this task.
7. Additional Test: assign tasks to several users from different teams. Check this volunteer account can UNASSIGN tasks from ONLY their TEAM members, not others through the Dashboard -> task -> red X. 

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/112589476/e151e6ab-9574-4ce6-af9e-c73a09fbb27a
